### PR TITLE
Feat slack plugin

### PR DESCRIPTION
**Pull request by benib**
This needs plugin options like this: 
```
{
  token: process.env.SLACKBOT_TOKEN,
  slashCommandToken: process.env.SLACK_SLASH_COMMAND_TOKEN,
  postScreenshots: true,
  qEditorBaseUrl: 'url-to-q-editor',
  notifications: [
    {
      events: ['item.new', 'item.update'],
      channel: 'slack-channel-where-the-bot-user-with-slackbot-token-is-member',
      propertyFilters: {
        publication: 'nzz',
      }
    }
  ]
}
```

### CODE REVIEW

File: plugins/slack/slash-command-routes.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+            }
+          },
+          tags: ['api']
+        },
+        handler: async (request, h) => {
+          if (request.payload.text === 'stats') {
+            const numberOfItemsPerDayResponse = await server.inject('/statistics/number-of-items-per-day/7');
+            const stats = numberOfItemsPerDayResponse.payload;
+
+            const numberOfNewItemsResponse = await server.inect(`/statistics/number-of-items/${Date.now() - (7 * 24 * 60 * 60 * 1000)}`);
```
Comment (fuenkchen) [inline plugins/slack/slash-command-routes.js]: `server.inject`

File: test/e2e-tests.js [hunk 2]
```diff
@@ -335,8 +373,28 @@ lab.experiment('core schema endpoints', () => {
...
+    const response = await server.inject('/statistics/number-of-items-per-day/7');
+    expect(response.statusCode).to.be.equal(200);
+    expect(response.result).to.be.an.array();
+    expect(response.result[0].value).to.be.equal(3);
+  })
+});
+
 lab.experiment('screenshot plugin', async () => {
-  await it('returnes a screenshot with correct cache-control headers', { timeout: 5000 }, async () => {
+  await it('returnes a screenshot with correct cache-control headers', { timeout: 5000, plan: 3 }, async () => {
```
Comment (fuenkchen) [inline test/e2e-tests.js]: `returns`

Comment (fuenkchen) [inline test/e2e-tests.js]: Would be nice to have tests for the slack plugin as well.

**Comment by benib** [comment]
I will clean this up and find a way to test it and then PR again.

**Pull request closed without merging**